xbmc/settings/MediaSettings.cpp

```diff
@@ -94,7 +94,7 @@ bool CMediaSettings::Load(const TiXmlNode *settings)
       scalingMethod = (int)VS_SCALINGMETHOD_LINEAR;
     m_defaultVideoSettings.m_ScalingMethod = (ESCALINGMETHOD)scalingMethod;
 
-    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeCustom);
+    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeZoom110Width);
```

Comment:
I'm not sure if you did that on purpose (or if that was a bug on our side) but this also exposes ViewModeStretch16x9Nonlin on our gui. Which wasn't the case before, if I'm reading it right.